src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -305,16 +321,6 @@ public function getLogoutUrl()
         return $this->data['logout_url'];
     }
 
-    /**
-     * Returns the FQCN of the security voters enabled in the application.
-     *
-     * @return string[]
-     */
-    public function getVoters()
```

Comment:
Removing the method is a BC break I'd try to avoid.

Reply:
@xabbuh I Restored the method with a deprecated flag